timeseries/src/main/scala/com/criteo/cuttle/timeseries/intervals/IntervalMap.scala

```diff
@@ -133,6 +133,10 @@ private[timeseries] object IntervalMap {
         m <- this.intersect(interval).toList
       } yield m): _*))
     }
+
```

Comment:
You are adding code you do not use, If you really feel this is something that needs to be done I would suggest you could move it to another PR.

Reply:
Cleaned it